src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -80,6 +80,9 @@ private sealed partial class TagSource : ForegroundThreadAffinitizedObject
             public event EventHandler Paused;
             public event EventHandler Resumed;
 
+            public TaggerDelay AddedTagNotificationDelay => _dataSource.AddedTagNotificationDelay;
```

Comment:
We generally have fields/simple-props before constructors.  Though we're not hugely consistent.